### DISCUSSION TITLE
How do i link my compiler to Ripes

### DISCUSSION THREAD

**Discussion by marquismarc0444** [mortbopet/Ripes#95] [Q&A]
HI my compiler is Clion and i dont know how to link it to the compiler path in ripes.

**Comment by mortbopet** [comment]
Hello @marquismarc0444
For building RISC-V programs, we need a compiler which is able to target the RISC-V instruction set - this is not the case for the built-in compiler shipped with CLion (which targets x86 processors).

You should follow the tutorial on [Building and Executing C programs with Ripes](https://github.com/mortbopet/Ripes/wiki/Building-and-Executing-C-programs-with-Ripes) which will refer you to a place where you can download a valid compiler.
Specifically, for windows, you need to download the `riscv64-unknown-elf-toolchain-...-x86_64-w64-mingw32` toolchain, if downloading a prebuilt toolchain from the sifive repository.
